babel/src/board.js

```diff
@@ -1,214 +1,13 @@
 'use strict';
 import * as _ from 'lodash';
+import iterchunks from './tools/iterchunks';
+import evaluate as affectOrbs.evaluate from './affectOrbs/evaluate';
+import unmatch as affectOrbs.unmatch from './affectOrbs/unmatch';
+// import './affectOrbs';
```

Comment:
I need some help determining how to do this. I want to have `evaluate.js` and `unmatch.js` live inside the `affectOrbs` directory. But the problem is that if I import them like that, there will be global `evaluate()` and `unmatch()` and also inside `Board` as methods.